scapy/contrib/isotp.uts

```diff
@@ -92,16 +106,19 @@ else:
 
 if "python_can" in CANSocket.__module__:
     import can as python_can
-    new_can_socket = lambda: CANSocket(iface=python_can.interface.Bus(bustype='socketcan', channel=iface, bitrate=250000))
+    new_can_socket0 = lambda: CANSocket(iface=python_can.interface.Bus(bustype='socketcan', channel=iface0, bitrate=250000))
+    new_can_socket1 = lambda: CANSocket(iface=python_can.interface.Bus(bustype='socketcan', channel=iface1, bitrate=250000))
 else:
-    new_can_socket = lambda: CANSocket(iface)
+    new_can_socket0 = lambda: CANSocket(iface0)
+    new_can_socket1 = lambda: CANSocket(iface1)
+
 
 print("CAN sockets should work now")
 
 
 # Verify that a CAN socket can be created and closed
 ~ conf linux needs_root
-s = new_can_socket()
+s = new_can_socket0()d
```

Comment:
@denis1011 This trailing 'd' let your tests fail.

Reply:
@polybassa  is removed thanks.